ulid_test.go

```diff
@@ -347,6 +347,20 @@ func TestTimestamp(t *testing.T) {
 	}
 }
 
+func TestTimestampToTime(t *testing.T) {
```

Comment:
Let’s also test the round tripping with `Timestamp`. Using `testing/quick` like the other tests.